examples/hello_world.c

```diff
@@ -0,0 +1,8 @@
+#include "../czech.h"
```

Comment:
Leftover from when you tested it?

Reply:
Well, if I remove that, it won't compile, right? I assumed most people wouldn't add czech.h to their include path.

Reply:
Yeah but wouldn't `./` be better than `../`?

Reply:
As far as I know, C/C++ import paths are relative to the currently processed file, in this case `./examples/hello_world.c`. So if I want to access `./czech.h`, then I need to go one directory up.

Just clone it and try it. It won't work unless it's in the same directory.

```
.\examples\hello_world.c:1:10: fatal error: './czech.h' file not found
#include "./czech.h"
         ^
1 error generated.
```

Reply:
Ah, my bad, I missed that the hello world is in a separate directory. Thought we'd put that in the README so that people can see the "art" we're creating

Reply:
I wanted to add more examples in the future, but I can copy it over to the README too if you want; of course with the correct include path.

Site note: I wish we could override `#include` as well, but that is likely impossible.